tests/unit/components/Projects/Projects.test.js

```diff
@@ -37,25 +37,25 @@ jest.mock( "@react-navigation/native", ( ) => {
   };
 } );
 
-const renderProjects = ( ) => render(
-  <NavigationContainer>
-    <Projects />
-  </NavigationContainer>
+const renderProjects = async () => waitFor(
```

Comment:
For a unit test, you probably don't need `waitFor` if you've mocked all your effects. If you're waiting for anything it should probably be an integration test because you're testing both the code under test _and_ whatever you're waiting for.